proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/route/RouteActivity.java

```diff
@@ -83,9 +83,9 @@ public CompletableFuture<QueryRouteResponse> queryRoute(ProxyContext ctx, QueryR
             }
 
             QueryRouteResponse response = QueryRouteResponse.newBuilder()
-                .setStatus(ResponseBuilder.getInstance().buildStatus(Code.OK, Code.OK.name()))
-                .addAllMessageQueues(messageQueueList)
-                .build();
+                    .setStatus(ResponseBuilder.getInstance().buildStatus(Code.OK, Code.OK.name()))
+                    .addAllMessageQueues(messageQueueList)
+                    .build();
```

Comment:
It seems that there is a code style issue. Try importing the  [rmq_code_style](https://github.com/apache/rocketmq/blob/develop/style/rmq_codeStyle.xml) to avoid introduce unnecessary code changes.

Reply:
fix done